tests/integration/test_able_to_sent_to_commands.py

```diff
@@ -0,0 +1,190 @@
+import time
+import json
+import random
+
+import pytest
+
+import paho.mqtt.client as mqtt
+from pymodbus.client import ModbusTcpClient
+from pymodbus.payload import BinaryPayloadDecoder
+from pymodbus.constants import Endian
+
+def test_able_to_update_coil():
+    random_value = random.randint(1, 2) % 2 == 0
+
+    message_dictionary = {
+        "action": "testRegister",
+        "value": random_value
+    }
+    message = json.dumps(message_dictionary)
+
+    mqttClient = mqtt.Client()
+    modbusClient = ModbusTcpClient(
+        "localhost",
+        5020,
+    )
+
+
+    modbusClient.connect()
+    mqttClient.connect("localhost", 1883, 60)
+
+    mqttClient.publish("commands/test", message)
+
+    time.sleep(5)
```

Comment:
this is a long time to sleep in the context of automated tests, is it necessary for it to be that long?

Reply:
I would recommend to use a different name here. This test, rather than being an integration test, is more of an end-to-end or functional test. It is testing the whole thing end to end. In such context, the amount of tests should be minimal (probably less than 10 tests in total).
As such, having tests that wait seconds, is not an issue.

But again, I recommend to use a different name here, as typically, integration is more about checking the integration point, but not necessarily the entire end-to-end flow. Happy with either `end-to-end` or `functional` folder to differentiate them.